### DISCUSSION TITLE
Define name of a new created column

### DISCUSSION THREAD

**Discussion by BertrandBorel** [h2oai/datatable#3489] [Q&A] [answered]
Hello,

I search to define the name of my new transformed column.

Here an example : 

`DT[[f.FIRSTNAME.re_match("Jean.+")] and [f.AGE > 80 ], [f.FIRSTNAME,f.NAME, f.SEXE, ifelse(f.SEX == 1,'Male','Female') ,f.AGE] ] `

By default, the `ifelse(f.SEX == 1,'Male','Female)` function will return a column name **"C0"**.

Is possible to do something similar to rename it directly like this : 

`DT[[f.FIRSTNAME.re_match("Jean.+")] and [f.AGE > 80 ], [f.FIRSTNAME,f.FIRSTNAME, f.SEX, ` **f.NEWCOL=ifelse(f.SEX == 1,'Male','Female')** `,f.AGE] ] `

Thanks for your help.

**Comment by lmullany** [comment, marked as answer]
You may use `alias`:

```
from datatable import dt, f
DT = dt.Frame([[1, 2, 3], ["one", "two", "three"]])
DT.names = ['A','B']
DT[f.A >1,[f.A, f.B, dt.ifelse(f.B=="three",1,0).alias("C")]]
```

Output:
```
   |     A  B          C
   | int32  str32  int32
-- + -----  -----  -----
 0 |     2  two        0
 1 |     3  three      1
```

**Reply by BertrandBorel** [reply]
It works perfectly ! Thanks a lot for your help !
